editor/gui/editor_run_bar.cpp

```diff
@@ -465,25 +503,55 @@ EditorRunBar::EditorRunBar() {
 	main_hbox->add_child(run_native);
 	run_native->connect("native_run", callable_mp(this, &EditorRunBar::_run_native));
 
-	play_scene_button = memnew(Button);
+	bool add_play_xr_mode_options = false;
+#ifndef _3D_DISABLED
+	if (OS::get_singleton()->has_feature("xr_editor") &&
+			(XRServer::get_xr_mode() == XRServer::XRMODE_ON ||
+					(XRServer::get_xr_mode() == XRServer::XRMODE_DEFAULT && GLOBAL_GET("xr/openxr/enabled")))) {
+		// If this is the XR editor and openxr is enabled, we turn the `play_scene_button` and
+		// `play_custom_scene_button` into MenuButtons to provide the option to start a scene in
+		// either regular mode or XR mode.
+		add_play_xr_mode_options = true;
+	}
+#endif // _3D_DISABLED
+
+	if (add_play_xr_mode_options) {
+		MenuButton *menu_button = memnew(MenuButton);
+		PopupMenu *popup = menu_button->get_popup();
+		popup->add_item(TTRC("Run Current Scene in Regular mode"), 0);
+		popup->add_item(TTRC("Run Current Scene in XR mode"), 1);
```

Comment:
```suggestion
		popup->add_item(TTRC("Run in Regular mode"), 0);
		popup->add_item(TTRC("Run in XR mode"), 1);
```
I think, labels are a bit long. It would be better to shorten them.

Reply:
I agree that these are kind of long, although, I don't know about throwing away "Scene" altogether, so folks don't think this going to run the full app with its default scene.

Maybe `"Run Scene in Regular Mode"` (dropping `"Current"`)?